idaes/models/unit_models/gibbs_reactor.py

```diff
@@ -248,7 +262,7 @@ def gibbs_minimization(b, t, p, j):
                     * b.control_volume.properties_out[t].config.parameters.element_comp[
```

Comment:
Would you recommend changing the code here to `properties_out[t].params` instead of `properties_out[t].config.parameters`. This is for the uniformity.